### ISSUE TITLE
Command line options

### ISSUE THREAD

**Issue by Gavin-Holt** [mougino/MaNo#1]
Hi,

Mano is really nice, clean and quick. Much lighter than other Markdown editors. 

I intend to use Mano to learn Markdown, which flavour does Mano produce?

Are there any command line options? It would be useful to automatically parse a Markdown file on opening and display in the browser. This would replace my current script/hack for viewing Markdown on local windows file system.

Many thanks

Gavin

**Comment by mougino** [comment]
Hi Gavin, first of all thank you for your interest.

MaNo produces [GitHub Markdown](https://www.markdownguide.org/tools/github-pages/) (maybe an older version since I started writing the program in 2017).

There is no command line option for the moment as MaNo is a pure Win32 GUI program. But 'hacking' the executable to make it console-compatible is doable (set byte #221 to 0x03 from memory).

I think I could pull a small change. What syntax do you propose for this command-line conversion + browser launch?

**Comment by mougino** [comment]
Link is incorrect in my previous comment: see [Mastering Markdown (github.com)](https://guides.github.com/features/mastering-markdown/) instead.

**Comment by Gavin-Holt** [comment]
Hi,

How about 

`Mano -v filename` to parse save, load in browser and exit

`Mano.exe -i filename.md -o filename.html ` batch convertion.

Kind regards

Gavin

**Comment by mougino** [comment]
What should the program do if a target file already exists? should it overwrite it without warning, cancel and do nothing, or ask you what to do (by console prompt) ?

**Comment by mougino** [comment]
You can find a test version at http://mougino.free.fr/tmp/MaNo1.2.0.zip

The program works as before in graphical mode, but it is also now possible to use it in batch mode via command line.
For a list of options, type `mano /?` or `mano -h` at the prompt.

By default, the target html is always overwritten.

**Comment by Gavin-Holt** [comment]
Hi

Flawless - many thanks. For others the chosen syntax is :
```
Usage: mano -c[v] input.md
       -c  = convert input.md to input.htm
       -cv = convert + view
```

I have starred this project and added a link in [stackoverflow](https://stackoverflow.com/questions/12931966/github-readme-md-windows-editor/69867109#69867109).

Kind Regards Gavin Holt

**Comment by mougino** [comment]
Ticket should not be closed, only test version was provided.

Reopening to commit changes and officialize new version.

**Comment by mougino** [comment]
Fixed in 1.2.0, recent changes committed.
Updated installer+zip available at http://mougino.free.fr/mano.html
